### PR TITLE
Restore js (security) tests

### PR DESCRIPTION
**Pull request by mroest**
The current travis build did not work since yarn tests/checks are not
running. This updates the js/css depencencies and restores the travis
build.

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -2,9 +2,12 @@
 {
   "compilerOptions": {
     /* Basic Options */
-    "target": "es3",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
```
Comment (MKodde) [inline tsconfig.json:5]: Switching to ES5 might 'screw' some older browsers. Is there a clear benefit we should move to ES5?

SIde note. We do not actively support IE < 10 browsers, and for other modern browsers: FF, Chrome, Safari we support the current and previous version. However. If without too much effort we can support other/older browsers. We should never refrain from doing so. As some institutions still use ancient browsers and supporting them with _minimal_ effort should always be attempted. And, we should never put our-self at risk by supporting older browsers.

File: webpack.config.js [hunk 2]
```diff
@@ -6,52 +6,37 @@ Encore
     .cleanupOutputBeforeBuild()
     // Convert typescript files.
     .enableTypeScriptLoader()
-    .autoProvidejQuery()
```
Comment (MKodde) [inline webpack.config.js:9]: No issues with removing this? :+1:

File: webpack.config.js [hunk 3]
```diff
@@ -6,52 +6,37 @@ Encore
...
-        options.compress = {
-            screw_ie8: false,
-        };
-        options.mangleProperties = {
-            screw_ie8: false,
-        };
-        options.output = {
-            screw_ie8: false,
-        };
-    })
```
Comment (MKodde) [inline webpack.config.js:26]: See my previous comment about browser support.

**Review by MKodde — COMMENTED** [review]
Nice work again @mroest! This PR seems like a classic case of rabbit hole. Maybe next time try to bundle your work more atomically in several commits. I'm fine with these kind of PR's but I'd like the git history to be a bit more fine grained than this.

Also, see my comments below.

File: tsconfig.json (hunk 1 above)
Comment (mroest) [reply tsconfig.json:5]: > Switching to ES5 might 'screw' some older browsers. Is there a clear benefit we should move to ES5?
Projects like Webauthn also use es5. ES5 is already more than 10 years old. Older browser are not safe anyway so i think we should not support that. Besides that, Engineblock has support for IE10+ which is similar as for ES5.

File: webpack.config.js (hunk 2 above)
Comment (mroest) [reply webpack.config.js:9]: I guess the diff is a bit weird but, autoProvideJQuery() is still there. Its moved a bit down in the file.

**Comment by mroest** [comment]
> Nice work again @mroest! This PR seems like a classic case of rabbit hole. Maybe next time try to bundle your work more atomically in several commits. I'm fine with these kind of PR's but I'd like the git history to be a bit more fine grained than this.

Yes, agreed. It turned out in to something bigger than expected. Although all the changes is just to get the travis build working it maybe would have been better to al least have it in a couple of different commits.

File: tsconfig.json (hunk 1 above)
Comment (MKodde) [reply tsconfig.json:5]: Thanks for verifying this!

**Review by MKodde — APPROVED** [review]

**Pull request merged**